### PR TITLE
BLD: added cibuildwheel to pipeline

### PR DESCRIPTION
**Pull request by tla256**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -56,3 +58,38 @@ jobs:
...
         tox -e py36
         codecov
+
+- job: Linux_Build
+  pool: {vmImage: 'Ubuntu-16.04'}
+  steps: 
+    - task: UsePythonVersion@0
+    - bash: |
+        python -m pip install --upgrade pip
+        python -m pip install tox pytest click cython numpy pandas plotly scitrack tqdm tinydb cibuildwheel==0.11.1 pytest-azurepipelines
```
Comment (GavinHuttley) [inline azure-pipelines.yml]: Can we change these massive long lines to `python -m pip install .[dev] cibuildwheel==0.11.1 pytest-azurepipelines`?

That should trigger install of the dev dependencies (and thus it's an implicit test we haven't forgotten something) while also adding the other just for CI dependencies.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -56,3 +58,38 @@ jobs:
...
+        cibuildwheel --output-dir wheelhouse .
+    - task: PublishBuildArtifacts@1
+      inputs: {pathtoPublish: 'wheelhouse'}
+- job: MacOS_Build
+  pool: {vmImage: 'macOS-10.13'}
+  steps: 
+    - task: UsePythonVersion@0
+    - bash: |
+        python -m pip install --upgrade pip
+        python -m pip install tox pytest click cython numpy pandas plotly scitrack tqdm tinydb cibuildwheel==0.11.1
```
Comment (GavinHuttley) [inline azure-pipelines.yml]: as above

File: azure-pipelines.yml [hunk 3]
```diff
@@ -56,3 +58,38 @@ jobs:
...
+- job: Windows_Build
+  pool: {vmImage: 'vs2017-win2016'}
+  steps:
+    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.6', architecture: x86}}
+    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.6', architecture: x64}}
+    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.7', architecture: x86}}
+    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.7', architecture: x64}}
+    - bash: |
+        python -m pip install --upgrade pip
+        python -m pip install tox pytest click cython numpy pandas plotly scitrack tqdm tinydb cibuildwheel==0.11.1
```
Comment (GavinHuttley) [inline azure-pipelines.yml]: as above

File: azure-pipelines.yml (hunk 1 above)
Comment (tla256) [reply azure-pipelines.yml]: I see that `pytest-azurepipelines` is already in `[dev]`, so I will omit that, but the change has been made, will push shortly.

**Pull request merged**